### PR TITLE
Fix closest

### PR DESCRIPTION
**Pull request by Manosfrias**
Fix closest bug and delegate

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,5 +3,6 @@ npm-debug.log
 .idea
 test/domassist.test.dist.js
 dist
+packag-lock.json
```
Comment (Antonio-Laguna) [inline .gitignore]: Typo here

File: package.json [hunk 2]
```diff
@@ -31,9 +31,9 @@
     "eslint-plugin-compat": "^2.5.1",
     "eslint-plugin-import": "^2.14.0",
     "phantomjs-prebuilt": "^2.1.16",
-    "scriptkit": "0.4.0",
+    "scriptkit": "^1.0.0",
```
Comment (Antonio-Laguna) [inline package.json:34]: Nice touch updating this!

**Review by Antonio-Laguna — CHANGES_REQUESTED** [review]
Nicely done! Thanks for separating the test into its own file and updating Scriptkit too! Just a minor typo found

File: package.json (hunk 2 above)
Comment (Manosfrias) [reply package.json:34]: Thank you!

**Review by Antonio-Laguna — APPROVED** [review]

**Pull request merged**